ansible/roles/ci/deploy/build_debs/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - { role: ci/common/packages }
```

Comment:
I have issues with such approach in the past. Somehow this doesn't influenced sequence of the execution or similar error. Solution which I have come up with is to put tag for dependent modules into docker.yml for ci/common/packages as well.